resources/public/api/conf/1.0/scenarios/submitBusEOPS_scenarios.md

```diff
@@ -67,5 +67,8 @@
             <td><p>RULE_NON_MATCHING_PERIOD</p></td>
             <td><p>Simulate scenario where an End of Period Statement without a matching accounting period cannot be submitted.</p></td>
         </tr>
+            <td><p>BUSINESS_EOPS_RULE_ALREADY_SUBMITTED</p></td>
+            <td><p>Simulate scenario where an End of Period Statement has already been submitted for the period supplied.</p></td>
+        </tr>
```

Comment:
There is no gov test scenario in the stub called `BUSINESS_EOPS_RULE_ALREADY_SUBMITTED` I believe this scenario is already covered by the `RULE_ALREADY_SUBMITTED`scenario